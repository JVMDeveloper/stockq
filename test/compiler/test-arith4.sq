print(4.4 + 7);
print(19 - 3.0);
print(4 * 5.2);
print(8/5.5);
print(7%2.2);

