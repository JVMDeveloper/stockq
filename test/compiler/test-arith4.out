11.400000
16.000000
20.800000
1.454545
0.400000

